src/05_lists.py

```diff
@@ -8,22 +8,28 @@
 
 # Change x so that it is [1, 2, 3, 4]
 # YOUR CODE HERE
+x.append(4)
 print(x)
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
 # YOUR CODE HERE
+x += y
```

Comment:
This will work in most situations but `+=` only works with locally and globally scoped variables. So if `x` was defined in an outer function and then reassigned with `+=` in an inner function it wouldn't work even with the `nonlocal` keyword. Check out the `extend()` method.